src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -58,6 +58,53 @@ public void GetProcessesByName_RemoteMachineNameUnix_ThrowsPlatformNotSupportedE
             Assert.Throws<PlatformNotSupportedException>(() => Process.GetProcessesByName(currentProcess.ProcessName, machineName));
         }
 
+        [Fact]
+        public void GetProcesses_LongProcessName()
+        {
+            string commandName = "sleep";
+
+            // sleep program doesn't exist on some distros
+            if (IsProgramInstalled(commandName))
```

Comment:
If you like you can return here if not installed. Early return reduces indentation which can make it easier to read the rest of the method

Reply:
ok